flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/log/CygnusLogger.java

```diff
@@ -40,7 +48,14 @@ public CygnusLogger(String name) {
      * @param message
      */
     public void info(String componentName, String message) {
-        info("[" + componentName + "] " + message);
+        try {
+            info("[" + componentName + "] " + message);
+        } catch (Exception e) {
+            if (shutdown) {
+                System.err.println("A problem with the logging system was found... shutting down Cygnus right now!");
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/083ba577850ad21c798abe1c183f794ee2f2c4ea